hibernate-core/src/main/java/org/hibernate/sql/model/ast/ColumnValueBindingList.java

```diff
@@ -54,7 +54,7 @@ public void addRestriction(String columnName, String columnWriteFragment, JdbcMa
 		add( createValueBinding( columnName, columnWriteFragment, jdbcMapping ) );
 	}
 
-	protected ColumnValueBinding createValueBinding(
+	public ColumnValueBinding createValueBinding(
```

Comment:
I think the following change might be more appropriate, given that `createValueBinding` will cause side effects.

```suggestion
	public boolean containsColumn(String columnName, JdbcMapping jdbcMapping) {
		return contains( new ColumnReference( mutatingTable, columnName, jdbcMapping ) );
	}
	
	protected ColumnValueBinding createValueBinding(
```